### PR TITLE
dev

### PR DESCRIPTION
**Pull request by JULIETYCL**
added function to modify default url

### CODE REVIEW

**Comment by AlexanderSenf** [comment]
Thank you for the update! I will test it as soon as I can.

Just one question - you haven't addressed the comment regarding the README file yet. Could you please ensure that the README doesn't need any further changes as well?

**Comment by AlexanderSenf** [comment]
Actually, the bit about the options that I praised in the previous comment has disappeared. Please put it back, it was good. I will not approve without this,

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Consequently, the base URL would change to
 
 4. Once the server is up and running, you can start using the app. For ease of testing, follow these steps: 
 
-(1) post call: `python3 app.py post` (this call will return a drs object id, e.g. 12345)
+(1) post call: `action post` (this call will return a drs object id, e.g. 12345)
```
[the remark this answers is not in the document]
Comment (JULIETYCL) [reply README.md]: Noted. I will update it

**Review by AlexanderSenf — APPROVED** [review]
Thank you for the changes! LGTM

**Review by AlexanderSenf — APPROVED** [review]
LGTM

**Pull request merged**